README.md

```diff
@@ -149,24 +128,52 @@ should be used that bootstraps the server. It installs Ansible's dependencies,
 and sets up a non-root user for Ansible to use. A playbook should be applied to
 a host by means of a wrapper script around `ansible-playbook`, that posts
 progress notifications to the committee's Slack team, among a few other things.
-To bootstrap the new server, the script should be run as follows:
-`$ ansible/scripts/run-playbook.sh <ENVIRONMENT>
-ansible/bootstrap-new-host.yml`, where `<ENVIRONMENT>` should be substituted by
-either `production` or `staging`.
 
 After the bootstrapping, the main playbook can be run to completely set up the
 server. The main playbook can be applied in the same way as the bootstrap
 playbook.
 
 When this has successfully finished, a server exists that matches one of the
-environments. When migrating the production server, a few more tasks should be
-performed, which is explained in detail in [this guide][deployment-new-production].
+environments.
+
+### Step-by-step guide
+These are the steps to follow to set up a new development or production server.
+Some of the steps require you to specify which of the two you are setting up.
+
+If you want to migrate from an existing server, a few additional tasks should be
+performed, which are explained in detail in [this guide][deployment-new-production].
+
+##### On Digital Ocean:
+1. Create a droplet named either `dev.svsticky.nl` (staging) or `svsticky.nl` (production).
+1. Assign a floating IP to the new droplet.
+
+
+##### On your local terminal:
+1. Download the repository and enter the folder.
+`$ git clone https://github.com/svsticky/sadserver`
+`$ cd sadserver`
+
+1. Update the submodule and enter the ansible folder.
+`$ git submodule update --init`
+`$ cd ansible`
+
+1. Bootstrap the host for either production or staging.
+`$ ./scripts/run-playbook.sh (production|staging) bootstrap-new-host.yml`
+You do not need to enter a SUDO password, but you do need to enter the correct Vault password.
+At the end of the process you will receive a newly generated SUDO password, which you will need in the next step.
+
+1. Run the main playbook for either production or staging.
+`$ ./scripts/run-playbook.sh (production|staging) main.yml`
+
+1. If you are not migrating from a previous version of the server, you will need to run these two commands to enable Koala.
```

Comment:
I think `start.yml` also needs to be run when migrating from a previous server. It just needs to be done *after* you've manually migrated the db.